### PR TITLE
hh trying to merge

### PR DESCRIPTION
**Pull request by HakoTheStreamer**
hako is asking pull request after making changed to readme. md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # testTempGitSocial
 
 To test and practice Git socials First commit by nzrdev. after commit uploading
```
Comment (nzrdev) [inline README.md:3]: nice change 1

File: README.md [hunk 2]
```diff
@@ -5,3 +5,5 @@ To test and practice Git socials First commit by nzrdev. after commit uploading
 # This line is from feature-readme and Hako is making changes.
 
 ## further commits will appear in pull request from my branch -"Hako"
+
+### It is third change though -HAKO-feature reade.
```
Comment (nzrdev) [inline README.md:9]: Third one?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
 # testTempGitSocial
 
 To test and practice Git socials First commit by nzrdev. after commit uploading
+
+# This line is from feature-readme and Hako is making changes.
```
Comment (nzrdev) [inline README.md:5]: I think this one is from 0

File: README.md [hunk 4]
```diff
@@ -1,3 +1,9 @@
 # testTempGitSocial
```
Comment (nzrdev) [inline README.md:1]: aasdasd

File: README.md (hunk 2 above)
Comment (HakoTheStreamer) [reply README.md:9]: Third one?Third one?

**Review by HakoTheStreamer — COMMENTED** [review]
ok. trying review changes

**Review by nzrdev — COMMENTED** [review]
looks fine. -nzrdev -review changesdan

**Review by HakoTheStreamer — COMMENTED** [review]
hako-trying to comment after view checked. and seeing approve not possible by Hako

**Review by nzrdev — CHANGES_REQUESTED** [review]
duypde comment goyyan VSCODE icinde

**Review by HakoTheStreamer — COMMENTED** [review]
request change reply etmeye calisyorum

**Review by nzrdev — APPROVED** [review]

**Pull request merged**